docs/sources/reference/api/docker_remote_api.md

```diff
@@ -71,15 +71,33 @@ This endpoint now returns `SystemTime`, `HttpProxy`,`HttpsProxy` and `NoProxy`.
 
 ### What's new
 
+**New!**
+Build now has support for `LABEL` command which can be used to add user data
+to an image.  For example you could add data describing the content of an image.
+
+`LABEL "com.example.vendor"="ACME Incorporated"`
+
+**New!**
 `POST /containers/(id)/attach` and `POST /exec/(id)/start`
 
 **New!**
 Docker client now hints potential proxies about connection hijacking using HTTP Upgrade headers.
 
+`POST /containers/create`
+
+**New!**
+You can set labels on container create describing the container.
```

Comment:
"fast lookup through the API" could presumably be added for env vars. So really the key difference is in the visibility to containers. A sentence somewhere along the lines of "Unlike env vars, labels are not visible to processes running inside a containers" would be great. Though quite where that would go I do not know. Perhaps @moxiegirl can advise.